### PR TITLE
SLVS-1573 Update product name in user-facing labels

### PR DESCRIPTION
**Pull request by gabriela-trutan-sonarsource**
[SLVS-1573](https://sonarsource.atlassian.net/browse/SLVS-1573)

[SLVS-1573]: https://sonarsource.atlassian.net/browse/SLVS-1573?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Integration.Vsix/Resources/Strings.Designer.cs [hunk 1]
```diff
@@ -500,7 +500,7 @@ public static string ToolsOptions_ErrorOpeningSettingsFile {
         ///      &quot;language&quot;: &quot;cpp&quot;
         ///    },
         ///    {
-        ///      &quot;language&quot; [rest of string was truncated]&quot;;.
+        ///      &quot;langu [rest of string was truncated]&quot;;.
```
Comment (vnaskos-sonar) [inline src/Integration.Vsix/Resources/Strings.Designer.cs:503]: change by accident?

**Review by vnaskos-sonar — APPROVED** [review]
👍

File: src/Integration.Vsix/Resources/Strings.Designer.cs (hunk 1 above)
Comment (gabriela-trutan-sonarsource) [reply src/Integration.Vsix/Resources/Strings.Designer.cs:503]: Oh, yes. Thank you for the catch!

Comment (gabriela-trutan-sonarsource) [reply src/Integration.Vsix/Resources/Strings.Designer.cs:503]: Ah, no. That's the designer.cs, which is generated

**Pull request merged**